lib/cdo/delete_accounts_helper.rb

```diff
@@ -141,9 +159,25 @@ def clean_and_destroy_pd_content(user_id, user_email)
     Pd::Application::Email.where(to: user_email).destroy_all if user_email.present?
 
     unless application_ids.empty?
-      # Pd::FitWeekend1819Registration does not inherit from Pd::FitWeekendRegistrationBase so both are needed here
-      Pd::FitWeekend1819Registration.where(pd_application_id: application_ids).update_all(form_data: '{}')
-      Pd::FitWeekendRegistrationBase.where(pd_application_id: application_ids).update_all(form_data: '{}')
+      # SQL query to anonymize Pd::FitWeekend1819Registration because the model no longer exists
+      application_ids.each do |app_id|
+        ActiveRecord::Base.connection.exec_query(
+          <<-SQL
+          UPDATE `pd_fit_weekend1819_registrations`
+          SET `pd_fit_weekend1819_registrations`.`form_data` = ''
+          WHERE `pd_fit_weekend1819_registrations`.`pd_application_id` = #{app_id}
+        SQL
+        )
+
+        # SQL query to anonymize Pd::FitWeekendRegistrationBase because the model no longer exists
+        ActiveRecord::Base.connection.exec_query(
+          <<-SQL
+          UPDATE `pd_fit_weekend_registrations`
+          SET `pd_fit_weekend_registrations`.`form_data` = ''
+          WHERE `pd_fit_weekend_registrations`.`pd_application_id` = #{app_id}
+        SQL
+        )
+      end
```

Comment:
Same... encapsulate in methods.

Did you notice how much you increase the size of the method here?
If it was encapsulated, I could just read the logical steps and if I needed to verify the behavior of the method I go to it.

It can also help create test cases in the future for a single behavior instead of this whole method if needed.

Reply:
Good call with these! Will update shortly